app/assets/javascripts/application.js

```diff
@@ -12,16 +12,21 @@
 //= require angular-ui-bootstrap
 //= require angular-ui-bootstrap-tpls
 //= require angular-sanitize
+//= require angular-ui-router
+//= require angular-rails-templates
 //= require moment
 //= require moment-strftime/build/moment-strftime.min
 //= require moment-timezone
 //= require sprintf
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require miq_angular_configuration
+//= require_tree ./components/
+//= require_tree ./controllers/
 //= require_tree ./directives/
+//= require_tree ./factories/
```

Comment:
@AparnaKarve unless you need an actual factory, can you change it to a service instead? We should ideally stop using factories altogether, and move towards ES6 classes as services, so unless you *have* to introduce a factory... (besides, you can always do `app.service('foo', new FooFactory())` instead of `app.service('foo', FooService)`..)